Sources/Sentry/SentryPerformanceTracker.m

```diff
@@ -51,7 +51,9 @@ - (SentrySpanId *)startSpanWithName:(NSString *)name operation:(NSString *)opera
         newSpan = [activeSpan startChildWithOperation:operation description:name];
     } else {
         SentryTransactionContext *context =
-            [[SentryTransactionContext alloc] initWithName:name operation:operation];
+            [[SentryTransactionContext alloc] initWithName:name
+                                                nameSource:kSentryTransactionNameSourceComponent
```

Comment:
SentryPerformanceTracker is a generic tool to create a transaction hierarchy, I don't think we should make this hard coded. 
I think this should be a parameter, and we could have an overload that uses component as default.